src/Loggers/LogHistograms.jl

```diff
@@ -59,3 +59,14 @@ end
 ## Backward compatibility
 log_histogram(logger, name, value, step) =
     log_histogram(logger, name, value; step=step)
+
+
+# Forward
```

Comment:
What is this comment about?

Reply:
Just a leftover from copy-pasting... thanks for spotting it!